adafruit_pcf8574.py

```diff
@@ -90,7 +90,7 @@ def write_pin(self, pin: int, val: bool) -> None:
 
     def read_pin(self, pin: int) -> bool:
         """Read a single GPIO pin as high/pulled-up or driven low"""
-        return (self.read_gpio() >> pin) & 0x1
+        return bool((self.read_gpio() >> pin) & 0x1)
```

Comment:
I think this is a bit clearer

```suggestion
        return ((self.read_gpio() >> pin) & 0x1) == 1
```

Reply:
changed with latest commit, thank you!